proposals/(draft) 2023-02-upgrade-v2.0.6/README.md

```diff
@@ -0,0 +1,19 @@
+## Background
+This is a draft of our SW Upgrade proposal to do a software upgrade to v2.0.6 software tag of the Panacea codebase. The purpose of sharing this draft is to gather validators feedback before we submit the proposal.
+
+## Proposal
+Panacea team would like to propose that we upgrade our chain’s ibc-go from v2 to v4.3.0 (with cosmos-sdk v0.45.12 and informalsystems/tendermint v0.34.24) and remove CosmWasm from the chain.
+
+We believe that removing CosmWasm requires some explanation from our end whereas upgrading ibc-go is straightforward to most of you.
```

Comment:
it would be good to mention that ibc-go v2 is in the end of life: https://github.com/cosmos/ibc-go/blob/main/RELEASES.md#stable-release-policy.

Reply:
Done!